web/src/actions/auth.js

```diff
@@ -56,7 +69,7 @@ export const checkAuth = () => dispatch => {
     .get('/me')
     .then(req => {
       dispatch(completeAuthCheck(req.data));
-      dispatch(fetchApd());
+      dispatch(loadData(req.data.activities));
```

Comment:
If the user is logged in, instead of just loading their APDs, load _all_ relevant data.